### ISSUE TITLE
v2.10: fatal error: parser.h: No such file or directory

### ISSUE THREAD

**Issue by Snipon** [ascii-boxes/boxes#80]
A build file seems to be missing, `parser.h`, when I build from the AUR repo.

**To Reproduce**  
Steps to reproduce the behavior, including, but not limited to:
- Value of the environment variable `BOXES`
- How *boxes* is invoked, including input text, ideally as a command which can be copy&pasted, for example

      pamac build boxes

  If invoked via an editor plugin, give name of editor and plugin, and its configuration
- If a custom box design is involved, its configuration
- The output text produced by *boxes*

```
❯ pamac build boxes
Preparing...
Cloning boxes build files...
Checking boxes dependencies...
Resolving dependencies...
Checking inter-conflicts...

To build (1):
  boxes  2.1.0-2    AUR


Edit build files : [e] 
Apply transaction ? [e/y/N] y


Building boxes...
==> Making package: boxes 2.1.0-1 (2021-04-20T21:33:04 CEST)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Downloading v2.1.0.zip...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   122  100   122    0     0    559      0 --:--:-- --:--:-- --:--:--   559
100  214k    0  214k    0     0   343k      0 --:--:-- --:--:-- --:--:-- 1177k
==> Validating source files with md5sums...
    v2.1.0.zip ... Passed
==> Removing existing $srcdir/ directory...
==> Extracting sources...
  -> Extracting v2.1.0.zip with bsdtar
==> Starting build()...
mkdir out
sed -e 's/--BVERSION--/2.1.0/; s/--GLOBALCONF--/\/etc\/boxes-config/' doc/boxes.1.in > doc/boxes.1
| For compilation info please refer to the boxes compilation FAQ
sed -e 's/--BVERSION--/2.1.0 (3eb9daa)/; s/--GLOBALCONF--/\/etc\/boxes-config/' src/boxes.in.h > out/boxes.h
| at https://boxes.thomasjensen.com/docs/faq.html#q5
make -C src BOXES_PLATFORM=unix LEX=flex YACC=bison build
make[1]: Entering directory '/var/tmp/pamac-build-simon/boxes/src/boxes-2.1.0/src'
make -C ../out -f ../src/Makefile BOXES_PLATFORM=unix ALL_OBJ="parser.o lex.yy.o boxes.o cmdline.o discovery.o generate.o input.o list.o parsecode.o parsing.o query.o regulex.o remove.o shape.o tools.o unicode.o" \
    CFLAGS_ADDTL="-O " STRIP=true flags_unix boxes
make[2]: Entering directory '/var/tmp/pamac-build-simon/boxes/src/boxes-2.1.0/out'
make[2]: Nothing to be done for 'flags_unix'.
flex --header-file=lex.yy.h ../src/lexer.l
flex --header-file=lex.yy.h ../src/lexer.l
gcc -I. -I../src -Wall -W -O  -D_FORTIFY_SOURCE=2  -c -o boxes.o ../src/boxes.c
gcc -I. -I../src -Wall -W -O  -D_FORTIFY_SOURCE=2  -c -o cmdline.o ../src/cmdline.c
gcc -I. -I../src -Wall -W -O  -D_FORTIFY_SOURCE=2  -c -o discovery.o ../src/discovery.c
gcc -I. -I../src -Wall -W -O  -D_FORTIFY_SOURCE=2  -c -o generate.o ../src/generate.c
gcc -I. -I../src -Wall -W -O  -D_FORTIFY_SOURCE=2  -c -o input.o ../src/input.c
gcc -I. -I../src -Wall -W -O  -D_FORTIFY_SOURCE=2  -c -o list.o ../src/list.c
gcc -I. -I../src -Wall -W -O  -D_FORTIFY_SOURCE=2  -c -o query.o ../src/query.c
gcc -I. -I../src -Wall -W -O  -D_FORTIFY_SOURCE=2  -c -o regulex.o ../src/regulex.c
gcc -I. -I../src -Wall -W -O  -D_FORTIFY_SOURCE=2  -c -o remove.o ../src/remove.c
In file included from ../src/boxes.c:33:
../src/parsing.h:28:10: fatal error: parser.h: No such file or directory
   28 | #include "parser.h"
      |          ^~~~~~~~~~
compilation terminated.
make[2]: *** [<builtin>: boxes.o] Error 1
make[2]: *** Waiting for unfinished jobs....
In file included from ../src/list.c:31:
../src/parsing.h:28:10: fatal error: parser.h: No such file or directory
   28 | #include "parser.h"
      |          ^~~~~~~~~~
compilation terminated.
make[2]: *** [<builtin>: list.o] Error 1
make[2]: Leaving directory '/var/tmp/pamac-build-simon/boxes/src/boxes-2.1.0/out'
make[1]: *** [Makefile:52: build] Error 2
make[1]: Leaving directory '/var/tmp/pamac-build-simon/boxes/src/boxes-2.1.0/src'
make: *** [Makefile:41: build] Error 2
==> ERROR: A failure occurred in build().
    Aborting...
```

- The error message your are getting, if any

**Expected behavior**  
What you *expected* to happen, for example the text that you expected *boxes* to produce

**Screenshots**  
If possible, add screenshots to help explain your problem.

**Environment (please complete the following information):**  
- *Boxes* version, e.g. `v2.10 (6d663cc)`
- Your operating system and architecture, `Manjaro Linux x86_64`

**Comment by tsjensen** [comment]
Thanks for reporting this!

Just a few minutes ago, we fixed a very similar case for #79. Please try the latest master (meaning at commit [d1a1cf1580](https://github.com/ascii-boxes/boxes/tree/d1a1cf158016f24ee9d4785e3e07b82329964d0f)). I should hope your problem is solved, too.

**Comment by tsjensen** [comment]
Closing because of no further feedback.

I assume the issue is solved for you. Feel free to reopen and clarify if not.
